com.unity.netcode.gameobjects/Runtime/Interest/InterestNodeStatic.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+
+namespace Unity.Netcode.Interest
+{
+    public class InterestNodeStatic<TObject> : IInterestNode<TObject>
```

Comment:
May be nice to have some sort of comment was this actually is and does.

Reply:
Added some verbiage, but I do need to do another comment pass